nix/tools/style.nix

```diff
@@ -50,6 +52,43 @@ let
         ${git}/bin/git diff-index --exit-code HEAD -- '*.hs' '*.lhs' '*.nix' '*.py'
       '';
 
+  pylintConfig = writeText ".pylintrc" ''
+    [MESSAGES CONTROL]
+    # Enable all pylint messages
+    enable=all
+
+    # Silence these pylint messages
+    disable=line-too-long,
+            no-member,
+            redefined-outer-name,
+            unused-wildcard-import,
+            use-implicit-booleaness-not-comparison-to-string,
+            use-implicit-booleaness-not-comparison-to-zero,
+            wildcard-import
```

Comment:
> I have silenced some pylint errors that seemed unreasonable

Hm, I think it'd be better to just go with defaults. How bad are the diffs when these options are enabled? Maybe you can share some branches or PRs with the diffs for each of the options?

Also linking https://github.com/PostgREST/postgrest/issues/4144, I believe using `writers.writePython3Bin`on Nix already enables all the options? So that might be one reason to just use defaults.

I recall I frequently hit the `line-too-long` one.